reorder.py

```diff
@@ -327,9 +377,11 @@ def printcols(cols: List[str], header: Optional[str] = None) -> None:
         print(f"    {row}")
 
 
-@click.command(context_settings={"help_option_names": ["-h", "--help"]})
+@click.command(context_settings={"help_option_names": ["--help"]})
```

Comment:
Hmm, I can see removing the `-h` option so you can set the host. IIRC this is now the default behavior, could you remove the `context_settings` argument completely?